src/rpc/serializers.ts

```diff
@@ -107,7 +107,7 @@ export function normalizeContinueOverrides(overrides: types.ContinueOverrides):
   return {
     method: overrides.method,
     headers: overrides.headers ? headersObjectToArray(overrides.headers) : undefined,
-    postData: overrides.postData,
+    postData: helper.isString(overrides.postData) ? new Buffer(overrides.postData, 'binary') : overrides.postData,
```

Comment:
Shouldn't string be treated as 'utf8' instead?

Reply:
It should. Adding a test.